controllers/operator_controller_test.go

```diff
@@ -374,6 +397,7 @@ var _ = Describe("Reconcile Test", func() {
 					Message: "failed to unpack",
 					Reason:  rukpakv1alpha1.ReasonUnpackFailed,
 				})
+				bd.Status.ObservedGeneration = bd.GetGeneration()
```

Comment:
Oh yeah, now that I'm seeing this repeated multiple times, it definitely seems ripe for being inside a `BeforeEach` inside a `When("bd status is up-to-date")`